Docs/src/tutorials/basictutorials/code/basictutorial1fullsource.md

```diff
@@ -0,0 +1,151 @@
+# Basic Tutorial 1 - Full source code {#basictutorial1fullsource}
+
+## Header file
+
+```cpp
+
+This source file is a part of OGRE
+
+#ifndef SRC_TUTORIALAPPLICATION_HPP_
+#define SRC_TUTORIALAPPLICATION_HPP_
+
+#include <OGRE/Ogre.h>
+#include <OGRE/Bites/OgreApplicationContext.h>
```

Comment:
if you correctly use `${OGRE_INCLUDE_DIRS}` (see https://ogrecave.github.io/ogre/api/1.10/setup.html) these should be just `<Ogre.h>`, `<OgreApplicationContext.h>` etc.